src/components/replacements-config/ReplacementsConfigModal.tsx

```diff
@@ -77,14 +77,19 @@ const ReplacementsConfigModal = ({ open, onClose }: Props) => {
 		let paths: string[] = ['$datetime'];
 
 		replayList.forEach((replayItem, index) => {
-			paths.push(`/${index}`);
+			let name = '';
+			if (replayItem.name) {
+				name = replayItem.name;
+			} else {
+				name = `untitled${index}`;
+			}
+			paths.push(`/${name}`);
```

Comment:
Here

Reply:
But this variable is used below in map functions:

`.map(x => `/${name}/body${x}`);`

If we will push values instead of it, we'll have to reuse the code, isn't it?

Reply:
then we can replace it all with `const name = replayItem.name || `untitled${index}`;`